src/main/scala/Game.scala

```diff
@@ -0,0 +1,215 @@
+object Game extends App {
+  def rng(higher:Int, lower:Int):Int = {
+    var computerGuess = scala.util.Random
+    var computerGuessInt = computerGuess.nextInt(higher - lower)
+    return computerGuessInt
```

Comment:
we don't use return in scala

computerGuess.nextInt(higher - lower) just have this as the last line in the method and you don;t need computerGuessInt variable at all